cmd/all.go

```diff
@@ -1,23 +1,31 @@
 package cmd
 
 import (
+	"sync"
+
 	"github.com/ipfs/testground/pkg/engine"
 	"github.com/urfave/cli"
 )
 
 // _engine is the default engine shared by all commands.
-var _engine *engine.Engine = func() *engine.Engine {
-	e, err := engine.NewDefaultEngine()
-	if err != nil {
-		panic(err)
-	}
-	return e
-}()
+var (
+	_engine     *engine.Engine
+	_engineErr  error
+	_engineOnce sync.Once
+)
+
+func GetEngine() (*engine.Engine, error) {
+	_engineOnce.Do(func() {
+		_engine, _engineErr = engine.NewDefaultEngine()
+	})
+	return _engine, _engineErr
+}
```

Comment:
We can merge this separately and rebase: https://github.com/ipfs/testground/pull/168.

Reply:
Without it, I can't run the sidecar from within the docker container (without access to the source).